src/blueprints/tv2/factories/tv2-atem-video-mixer-timeline-factory.ts

```diff
@@ -0,0 +1,48 @@
+import { Tv2VideoMixerTimelineObjectFactory } from '../value-objects/factories/tv2-video-mixer-timeline-object-factory'
+import { Tv2DownstreamKeyer } from '../value-objects/tv2-studio-blueprint-configuration'
+import { AtemDownstreamKeyerTimelineObject, AtemType } from '../../timeline-state-resolver-types/atem-types'
+import { Tv2AtemLayer } from '../value-objects/tv2-layers'
+import { DeviceType } from '../../../model/enums/device-type'
+
+export class Tv2AtemVideoMixerTimelineFactory implements Tv2VideoMixerTimelineObjectFactory {
+  public createDownstreamKeyerTimelineObject(downstreamKeyer: Tv2DownstreamKeyer, onAir: boolean): AtemDownstreamKeyerTimelineObject {
+    const downstreamKeyerNumber: string = String(downstreamKeyer.Number + 1)
```

Comment:
Converting it to a string is unnecessary.

Reply:
Forgot to remove the convertion, and update type, when what caused the need was removed.
Fixed now.